testtools/testcase.py

```diff
@@ -497,6 +492,23 @@ def assertThat(self, matchee, matcher, message='', verbose=False):
         if mismatch_error is not None:
             raise mismatch_error
 
+    def assertItemsEqual(self, a, b, message=''):
```

Comment:
Is this needed in this form?

`assertItemsEqual` is available in Python 2.7 and is named `assertCountEqual` in Python 3.

https://docs.python.org/2/library/unittest.html

Can they be used directly? 

eg. 

```python
try:
    # Python 2.7
    from unittest import assertItemsEqual as assertCountEqual
except ImportError:
    # Python 3
    from unittest import assertCountEqual
```